mg_utils/wait_for_workers.py

```diff
@@ -0,0 +1,124 @@
+# Copyright (c) 2023-2024, NVIDIA CORPORATION.
```

Comment:
I dont want to break our current workflow but as a non blocking item can we file an issue for investigating this vs  `client.wait_for_workers` .  Lets file an issue and add a note here. 

```
https://distributed.dask.org/en/stable/api.html#distributed.Client.wait_for_workers
```

Reply:
https://github.com/rapidsai/cugraph/issues/4082